scripts/bitwarden.sh

```diff
@@ -40,7 +46,7 @@ KEYCONNECTORVERSION="1.0.0"
 
 echo "bitwarden.sh version $COREVERSION"
 docker --version
-docker-compose --version
+$dccmd --version
```

Comment:
`--version` is not supported by `docker compose`

```suggestion
if [[ "$dccmd" == "docker compose" ]]; then
    $dccmd version
else
    $dccmd --version
fi
```